docs/admin/searx.favicons.rst

```diff
@@ -0,0 +1,249 @@
+.. _favicons:
+
+========
+Favicons
+========
+
+.. sidebar:: warning
+
+   Don't activate the favacions before reading the documentation.
+
+.. contents::
+   :depth: 2
+   :local:
+   :backlinks: entry
+
+Activating the favicons in SearXNG is very easy, but this **generates a
+significantly higher load** in the communication between client and server and
+additional resources are used on the server.
+
+To mitigate these disadvantages, various methods have been implemented,
+including a *cache*.  The cache must be parameterized according to your own
+requirements and maintained regularly.
+
+To activate favicons in SearXNG's result list set a default favicons-resolver in
```

Comment:
Add comma:

To activate favicons in SearXNG's result list set a default favicons-resolver in

To activate favicons in SearXNG's result list, set a default favicons-resolver in